### DISCUSSION TITLE
关于OSC缩放问题

### DISCUSSION THREAD

**Discussion by sqq135** [hooke007/mpv_PlayKit#314] [mpv 通用]
实在是找不到，所以在这里提问，关于控制器缩放问题
![屏幕截图 2023-05-24 235302](https://github.com/hooke007/MPV_lazy/assets/78015121/2d9aebd3-45c9-4101-9016-d07d5b148b93)
。因为比较中意modernX的界面，但是不满意它移动鼠标就自动弹出界面，找到个符合要求的fork。https://github.com/zydezu/ModernX/blob/main/modernx.lua，但是界面缩放太小了（应该就是这样设置的），于是我在手册中找相关参数，应该是没找对，都不能调整，这些
![image](https://github.com/hooke007/MPV_lazy/assets/78015121/348b4d08-489f-41ea-b73c-7416ac01c766)

希望来个指路人解惑，顺便问下右键菜单是在哪里设置的？链接关键词解答就可以了。

**Comment by hooke007** [comment]
你搞错仓库了。脚本选项找对应的开发者
手册里的参数是给osc用的

mpv没有右键菜单，菜单是我仓库的脚本方案

**Reply by sqq135** [reply]
我意思是这个应该是有通用的设置项吧，这种设置大小的也许属于最基本的设置项，我就是没找到这些

**Reply by hooke007** [reply]
ui缩放是脚本选项，本身就不是通用设置

mpv的有一个全局缩放是 hidpi-window-scale

**Reply by sqq135** [reply]
因为我找到对应的代码，所以以为有，那就没办法了，这条需要删掉吗？

**Reply by hooke007** [reply]
为什么要删掉，作为踩坑记录也是不错的
